test/integration/choosePageErrors.js

```diff
@@ -0,0 +1,76 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const cheerio = require('cheerio');
+const constants = require('../../app/lib/constants');
+const server = require('../../server');
+const iExpect = require('../lib/expectations');
+
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+const chooseErrorRoute = `${constants.siteRoot}/location`;
+
+describe('Choose page with error', () => {
```

Comment:
The tests could be restructured to execute the request once for each of the 2 options of `16to24` and `over25`, using the awaited `res`, each assertion can still be within its own `it` block. It'll speed up the test execution and removes some duplication.

Reply:
Not sure about doing the requests with 2 different queries but restructured to make it dry-er.

Reply:
I've made the changes in PR #28